pmd-javascript/etc/grammar/Ecmascript5.jj

```diff
@@ -281,7 +281,8 @@ TOKEN :
     |
         [
           "$",
-          "_"
+          "_",
+          "@"
```

Comment:
I wouldn't add `@` as part of an identifier. According to the standard, it is separate - see https://arai-a.github.io/ecma262-compare/snapshot.html?pr=2417#sec-decorators
When keeping it separately, we could potentially identify duplicated identifier names.